docs/introduction/faq.md

```diff
@@ -0,0 +1,216 @@
+
+Filecoin creates a hyper-competitive market for data storage – there will be many miners offering many prices, rather than one fixed price on the network. We expect Filecoin’s permissionless model and low barriers to entry to result in some very efficient operations and low-priced storage, but it’s impossible to say what exact prices will be until the network is live.
+
+### How can I confirm the mainnet launch time is on track?
+
+As we announced in our [roadmap update](https://filecoin.io/blog/roadmap-update-june-2020/), we’re continuing to make good progress towards mainnet launch, aiming towards the end of our mainnet launch window. However, all dates are still best-effort optimistic estimates - our top priority is to launch a secure and successful network.
+
+### What happens to the existing content on IPFS once Filecoin launches? What if nodes continue to host content for free and undermine the Filecoin incentive layer?
+
+IPFS will continue to exist as it is, enhanced with Filecoin nodes. There are many use cases that require no financial incentive. Think of it like IPFS is HTTP and Filecoin is a storage cloud like S3 – only a fraction of IPFS content will be there.
+
+People with unused storage who want to earn monetary rewards, should pledge that storage to Filecoin, and clients who want guaranteed storage should store that data with Filecoin miners.
+
+## Miners
+
+### How big will the deals be that the bot tries to make with miners during Incentivized Testnet?
+
+Expect ranges from a few kilobytes up to full 32GiB sectors.
+
+### Will the 5PB big miner incentive be included or excluded in the 4M incentives?
+
+The mechanisms for the incentive competition are [here](https://filecoin.io/blog/announcing-testnet-incentives/).
+
+### Lotus or Go-Filecoin, which is better for miners?
+
+Lotus is our reference implementation, and getting more developer attention right now.
+
+### What is your recommendation on the right hardware to use?
+
+While the Filecoin team does not recommend any specific hardware configuration, [we did share some setups](https://filecoin.io/blog/announcing-testnet-incentives/#hardware) we’ve used for various types of testing. However, it is overwhelmingly likely that there are more efficient setups, and we strongly encourage miners to test and experiment to find the best combinations.
+
+### We are worried about the ability for our network to handle the additional overhead of running a Filecoin node and still provide fast services for our customers. What are the computational demands of a Lotus node? Are there any metrics for node performance given various requirements?
+
+As we approach launch, our Testground team is prioritizing getting metrics to understand the performance of Lotus nodes; we will share these as they are determined. We are also still tweaking node and network design parameters, so these numbers will continue to improve - they will likely be orders of magnitude better by launch.
```

Comment:
As we approach launch, our Testground team is prioritizing getting metrics to understand the performance of Lotus nodes; we will share these as they are determined. We are also still tweaking node and network design parameters, so these numbers will continue to improve significantly over the next few months.